src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement/Altinn.AccessManagement.csproj

```diff
@@ -6,6 +6,7 @@
     </PropertyGroup>
 
     <ItemGroup>
+    		<PackageReference Include="Blazored.LocalStorage" />
```

Comment:
Dette bruker vi da vel ikke noen av APIene?

Reply:
nei, dette skal bort. var egentlig bare for å kjøre en test :+1: